Engine/src/sysFrame/Scene/SceneManager.cpp

```
@@ -0,0 +1,50 @@
+#include "SceneManager.h"
+
+#include "AbstractSceneFactory.h"
+#include "BaseScene.h"
+#include "System/System.h"
+
+SceneManager::~SceneManager() {
+    scene_.reset();
+    next_.reset();
+}
+
+void SceneManager::Update() {
```

cppcheck (2.10-2)

[style] The function 'Update' is never used.

(unusedFunction)